tests/lustre/typechecking/shouldFail/test_assumption_out_curr_value.lus

```diff
@@ -0,0 +1,13 @@
+node a (x:int) returns (y:int)
+let
+   y = x;
+tel
+
+
+node c (x:int) returns (y: int);
+(*@contract
+   assume x = a(y);    -- case 1. not allowed as assumption should not depend on current value of y 
+*)
+let
+        y = 2 * x;
+tel;
```

Comment:
I think this test check the same issue that `tests/lustre/typechecking/shouldFail/test_out_param_in_contract_assume.lus`. Remove one.

Reply:
deleted `test_assumption_out_curr_value.lus`